src/index.html

```diff
@@ -11,7 +11,138 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+      onsubmit="onSubmit()">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="form__field">
+          <label for="Surname">Surname:</label>
+          <input type="text"
+            name="Surname"
+            id="Surname"
+            required
+            autocomplete="off">
```

Comment:
```suggestion
          <input 
            type="text"
            name="Surname"
            id="Surname"
            required
            autocomplete="off"
          >
```
Размещай открывающую и закрывающую скобки на одной линии